tests/test_schema_evolution.py

```diff
@@ -48,7 +48,7 @@ def avro_to_bytes_with_schema(avro_schema, avro_dict):
 def bytes_with_schema_to_avro(avro_read_schema, binary):
     with BytesIO(binary) as bytes_io:
         reader = fastavro.reader(bytes_io, avro_read_schema)
-        return next(reader)
+        return reader.next()
```

Comment:
I don't understand how this fails. In Python 3 we use `obj.__next__()` while in Python 2 it's `obj.next()`. The builtin `next` should talk care of this.

@kurtostfeld Can you shed some light on this?